packages/ui/src/pages/MyStudiesPage/MyStudiesPage.vue

```diff
@@ -36,15 +37,70 @@ const studies = data;
       <AppButton @click="mutate">+ New Study</AppButton>
     </div>
 
-    <div class="flex flex-col">
-      <MyStudiesItem
-        v-for="study in studies"
-        :id="study._id"
-        :key="study._id"
-        :name="study.name"
-        :description="study.description"
-        @deleted="refetch"
-      />
+    <div
+      class="mt-8 mb-3 flex text-neutral-600 text-lg font-normal leading-tight"
+    >
+      Recent
+    </div>
+
+    <div class="space-y-6">
+      <!-- RECENT STUDIES -->
+      <div class="overflow-y-auto">
+        <div class="flex gap-4">
+          <RecentStudiesItem
+            v-for="study in studies"
+            :id="study._id"
+            :key="study._id"
+            :name="study.name"
+            :description="study.description"
+            @deleted="refetch"
+          />
+        </div>
+      </div>
+      <!-- STUDY TABLE -->
+      <div class="rounded-xl border border-neutral-300 overflow-y-auto">
```

Comment:
Instead of using div for the table, could look into using HTML tables (https://www.w3schools.com/html/html_tables.asp) or Element Plus tables (https://element-plus.org/en-US/component/table.html#table)